mediawiki/LocalSettings.d/mardi_namespaces.php

```diff
@@ -82,7 +82,7 @@
 $wgNamespaceProtection[NS_TASK] = array( 'overwriteprofilepages' );
 
 
-$wgGroupPermissions['sysop']['overwriteprofilepages'] = true;
+$wgGroupPermissions['user']['overwriteprofilepages'] = true;
```

Comment:
OK. Note that this might cause frustration if people's work gets overwritten by the profile page creation script. So it might be better to just remove the namespaces from the protection where this is reasonable?